agenda-webapps/src/main/webapp/vue-app/agenda-admin-settings/components/AgendaAdminConnectorSettings.vue

```diff
@@ -123,6 +125,7 @@ export default {
           connector.apiKey = connectorObj && connectorObj.apiKey || '';
           connector.loading = false;
           connector.editing = false;
+          connector.isExchange = (connector.name === 'agenda.exchangeCalendar') ? true : false;
```

Comment:
You can base on another prop called oauth for example which allows to decide if the api key is editable or not. This will keep the framework extensible, avoid mentioning exchange in the agenda codebase and prepare the transfert to the agenda-connectors addon codebase